.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/roberthunt/projects/envs/recipebox-L9jI5seE/bin/python"
```

Comment:
As a general rule, try not to commit your editor settings in the repo, as you don't want other people who use your repo to deal with your editor settings :P